packages/arb-token-bridge-ui/src/components/common/SettingsDialog.tsx

```diff
@@ -81,27 +75,9 @@ export const SettingsDialog = () => {
       isOpen={settingsOpen}
       heading="Settings"
```

Comment:
<img width="891" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/5ba35907-b945-45e3-9771-f9856bad109f">
heading size is 28px on Figma but 22px here

Reply:
I'll change it to `text-2xl`. 28px looks very large and out of place